File: java/org/openqa/selenium/internal/seleniumemulation/MetaKeyUp.java
package org.openqa.selenium.internal.seleniumemulation;

import org.openqa.selenium.WebDriver;

public class MetaKeyUp extends SeleneseCommand<Void> {
  private final KeyState keyState;

  public MetaKeyUp(KeyState keyState) {
    this.keyState = keyState;
  }

  @Override
  protected Void handleSeleneseCommand(WebDriver driver, String locator, String value) {
    keyState.metaKeyDown = false;
    return null;
  }
}

File: dotnet/src/Selenium.WebDriverBackedSelenium/Internal/SeleniumEmulation/MetaKeyUp.cs
using System;
using System.Collections.Generic;
using System.Text;
using OpenQA.Selenium;

namespace Selenium.Internal.SeleniumEmulation
{
    internal class MetaKeyUp : SeleneseCommand
    {
        private KeyState keyState;

        public MetaKeyUp(KeyState keyState)
        {
            this.keyState = keyState;
        }

        protected override object HandleSeleneseCommand(IWebDriver driver, string locator, string value)
        {
            keyState.MetaKeyDown = false;
            return null;
        }
    }
}
